ember_debug/object-inspector.js

```diff
@@ -500,7 +500,7 @@ export default EmberObject.extend(PortMixin, {
       if (proto.hasOwnProperty('toString') && !proto.hasOwnProperty('hasOwnProperty')) {
         return proto.toString();
       }
-      if (name === 'Class' || name.startsWith('_')) {
+      if (name === 'Class' || name.startsWith('_') || name.length === 1) {
```

Comment:
What is the use case for `name.length === 1`?

Reply:
in prod build the class names are uglyfied to 1 character

Reply:
@patricklx wouldn't that not always be the case though? What if you have more classes than the 26 letters in the alphabet? Would it make it 2 characters? @pzuraq @rwjblue is there a more reliable check we can do here?

Reply:
mmm, right, it could happen. but I think it must be more than 26 classes in a module or in same scope?
But I guess we could set it to `length <= 3`? should be 17,576‬ different classes.

Reply:
I would like to hear from @pzuraq, @chancancode, or @rwjblue on if there is something less brittle we could do for this.

Reply:
but if we only use prototype lookup, it would show to the `toString` from mixins...
And only default class name will not show the `toString` for mixins

Reply:
From what I can see, `toString` from mixins should be applied to the class, and shouldn't belong to the mixin directly. I actually can't find a way to name mixins at all, which is a bit unfortunate, but that seems to be by design 😕